chat-examples-swift.xcodeproj/project.pbxproj

```diff
@@ -876,7 +878,7 @@
 				MTL_ENABLE_DEBUG_INFO = INCLUDE_SOURCE;
 				MTL_FAST_MATH = YES;
 				ONLY_ACTIVE_ARCH = YES;
-				PRODUCT_BUNDLE_IDENTIFIER = "com.pubnub.chat-example";
+				PRODUCT_BUNDLE_IDENTIFIER = "chat-example.animal-forest-chat";
```

Comment:
Bundle identifiers according to Apple docs usually reverse URI for company and product name... that is why there was `com.pubnub` (reverse from `pubnub.com`).

Reply:
Got it 👍 
In case it wasn't clear, this change was because when it is `com.pubnub` users outside of the PubNub developer organization cannot run the sample code.